libc-top-half/musl/src/thread/pthread_create.c

```diff
@@ -424,6 +446,16 @@ int __pthread_create(pthread_t *restrict res, const pthread_attr_t *restrict att
 			if (map == MAP_FAILED) goto fail;
 		}
 #else
+		/*
+		 * Process the free queue before allocationg a new one.
+		 * Hopefully it enables a reuse of the memory.
+		 *
+		 * Note: We can't perform a simple "handoff" becasue allocation
+		 * sizes might be different. (eg. the stack size might differ)
+		 */
```

Comment:
The comment style within this file seems to be:

```
/* first line
  * second line
  * last line */
```

Reply:
fixed